packages/components/src/components/Snackbar/Snackbar.tsx

```diff
@@ -30,6 +30,8 @@ type snackbarData = {
     messageA11y?: string
     /** action button onPress logic for "Try again" (isError=true) or "Undo" button */
     onActionPressed?: () => void
+    /** offset from bottom of screen. defaults to 50 in SnackbarProvider */
+    offset?: number
```

Comment:
Nitpicky, but suggest moving up to be alphabetical.

Reply:
Moved